README.md

```diff
@@ -7,15 +7,15 @@ This chart bootstraps a Timed application deployment on a [Kubernetes](https://k
 ## Prerequisites
 
 - Kubernetes 1.7+ with Beta APIs enabled
-- PV provisioner support in the underlying infrastructure
```

Comment:
Why was this removed? Is dynamic PV provisioning not required anymore?

Reply:
Ok, technically it's not required, but as the database chart will need it in the future we should keep it.

Reply:
Agree. Added again.